### PR TITLE
Add vulnerability scanner image

### PR DESCRIPTION
**Pull request by kubadz**
**Description**

Changes proposed in this pull request:

- add vulnerability scanner image

**Related issue(s)**

#607

### CODE REVIEW

File: prow/images/vulnerability-scanner/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Docker Image containing node and go environments and Snyk application installed
+
+## Overview
+
+This folder contains the Golang Buildpack with nodejs and Snyk image that is based on the Golang Buildpack image. Use it to perform vulnerability scans.
```
Comment (tomekpapiernik) [inline prow/images/vulnerability-scanner/README.md]:
```suggestion
This folder contains the Golang Buildpack with Node.js and Snyk image that is based on the Golang Buildpack image. Use it to perform vulnerability scans.
```

File: prow/images/vulnerability-scanner/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+The image consists of:
+
+- Go 1.11.4
+- Dep 0.5.0
+- Gcloud 219.0.1
+- Docker 18.06.1*
+- nodejs
```
Comment (tomekpapiernik) [inline prow/images/vulnerability-scanner/README.md]:
```suggestion
- Node.js
```

File: prow/images/vulnerability-scanner/README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+## Overview
+
+This folder contains the Golang Buildpack with nodejs and Snyk image that is based on the Golang Buildpack image. Use it to perform vulnerability scans.
+
+The image consists of:
+
+- Go 1.11.4
+- Dep 0.5.0
+- Gcloud 219.0.1
+- Docker 18.06.1*
```
Comment (tomekpapiernik) [inline prow/images/vulnerability-scanner/README.md]: What does the `*` mean? Why is it there?

File: prow/images/vulnerability-scanner/README.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+- nodejs
+- Snyk 1.134.2
+
+## Installation
+
+To build the Docker image, run this command:
```
Comment (tomekpapiernik) [inline prow/images/vulnerability-scanner/README.md]: Is this sufficient? If other teams are to use this tool then you probably should provide more details. If just runs on Prow then ignore this comment.

Comment (kubadz) [reply prow/images/vulnerability-scanner/README.md]: It runs only on Prow.

**Pull request merged**